zigpy/application.py

```diff
@@ -10,6 +10,7 @@
 import zigpy.appdb
 import zigpy.backups
 import zigpy.config as conf
+from zigpy.config import CONF_ADDITIONAL_ENDPOINTS
```

Comment:
`zigpy.config` is imported as `conf` on the line above, you don't need to add another one.